lib/factree/dsl.rb

```diff
@@ -0,0 +1,40 @@
+# Readable shortcuts to common functions.
+module Factree::DSL
+  # Creates a {Conclusion} to return from a decision proc.
+  #
+  # The conclusion has an associated value (can be anything.)
+  #
+  # @param [Object] value Any value, including nil
+  # @return [Conclusion]
+  def conclusion(value=nil)
```

Comment:
My understanding is asking for a fact will always return a Conclusion? A Conclusion is always returned with a nil value even If there is no answer available?